mycity/mycity/intents/food_truck_intent.py

```diff
@@ -0,0 +1,190 @@
+    url_params = {
+        "f": "json",
+        "outFields": "*",
+        "outSR": "4326",
+        "returnGeometry": "true",
+        "where": day_of_week
+    }
+    logger.debug("Requesting data from ArcGIS server!")
+    food_truck_data_result = requests.get(base_url, url_params)
+    response_code = food_truck_data_result.status_code
+    logger.debug("Got response code: " + str(response_code))
+    if response_code != requests.codes.ok:
+        logger.debug('HTTP Error: '.format(response_code))
+        return {}
+
+    print(food_truck_data_result)
+
+    # Generate unique list of food truck locations
+    # to send to the Google Maps API
+    trucks = food_truck_data_result.json()['features']
+    truck_unique_locations = []
+    for truck in trucks:
+        if truck["attributes"]["Loc"] not in truck_unique_locations:
+            truck_unique_locations.append([truck["geometry"],
+                                           truck["attributes"]["Truck"],
+                                           truck["attributes"]["Loc"],
+                                           truck["attributes"]["Title"],
+                                           truck["attributes"]["Start_time"],
+                                           truck["attributes"]["End_time"]])
+    return truck_unique_locations
+
+
+def get_nearby_food_trucks(mycity_request):
+    """
+    Gets food truck info near an address
+
+    :param mycity_request: MyCityRequestDataModel object
+    :return: MyCityResponseObject
+    """
+    mycity_response = MyCityResponseDataModel()
+
+    # Get current address location
+    if CURRENT_ADDRESS_KEY in mycity_request.session_attributes:
+        current_address = \
+            mycity_request.session_attributes[CURRENT_ADDRESS_KEY]
+
+        address_parser = StreetAddressParser()
+        a = address_parser.parse(current_address)
+        address = str(a['house']) + " " + str(a['street_name']) + " " \
+                  + str(a['street_type'])
+        zip_code = str(a["other"]).zfill(5) if a["other"] else None
+
+        zip_code_key = intent_constants.ZIP_CODE_KEY
+        if zip_code is None and zip_code_key in \
+                mycity_request.session_attributes:
+            zip_code = mycity_request.session_attributes[zip_code_key]
+
+        usr_addr = gis_utils.geocode_address(address)
+        truck_unique_locations = get_truck_locations()
+        nearby_food_trucks = []
+        try:
+            for location in truck_unique_locations:
+                truck_lat_lon = list(location[0].values())
+                dist = gis_utils.calculate_distance(usr_addr, truck_lat_lon)
+
+                if dist <= MILE_IN_KILOMETERS:
+                    nearby_food_trucks.append(location)
+                else:
+                    continue
+
+            count = len(nearby_food_trucks)
+            if count == 0:
+                mycity_response.output_speech = "I didn't find any food trucks!"
```

Comment:
Prefer factoring out this failure `output_speech` and the formatted strings below into the speech_constants file